tests/test_lca.py

```diff
@@ -857,11 +841,75 @@ def test_gather_equiv_results():
 
         assert '2.7 Mbp     100.0%' in out
         assert 'Shewanella baltica' in out
-        assert '(** 1 equal matches)'
+        assert '(** 1 equal matches)' in out
         assert ('OS223' in out) or ('OS185' in out)
 
         assert os.path.exists(lca_db)
 
         r = csv.DictReader(open(os.path.join(location, 'matches.csv')))
         row = next(r)
         assert row['n_equal_matches'] == '1'
+
+
+def test_gather_old_lca_db():
+    with utils.TempDirectory() as location:
+        query_sig = utils.get_test_data('47+63.fa.sig')
+        lca_db = utils.get_test_data('lca/old-db-format-1.json')
+
+        cmd = ['lca', 'gather', query_sig, lca_db]
+        status, out, err = utils.runscript('sourmash', cmd,
+                                           in_directory=location,
+                                           fail_ok=True)
+
+        print(cmd)
+        print(out)
+        print(err)
+        assert 'Error! This is an old-style LCA DB.' in err
+        assert status != 0
+
+
+@utils.in_tempdir
```

Comment:
hmm, this looks like `tmpdir` in pytest... https://github.com/dib-lab/sourmash/pull/557/files#diff-9402af88a46ea03a8087cfebdbdb762bR16

and `RunnerContext` looks pretty close to a pytext fixture too https://docs.pytest.org/en/latest/fixture.html#fixture

Reply:
I figured they would ;). But I'm happy with the functionality for now and not inclined to fix it; can be adjusted in the future, yah?